src/java.base/share/classes/sun/nio/ch/AsynchronousChannelGroupImpl.java

```diff
@@ -48,9 +44,8 @@ abstract class AsynchronousChannelGroupImpl
 {
     // number of internal threads handling I/O events when using an unbounded
     // thread pool. Internal threads do not dispatch to completion handlers.
-    @SuppressWarnings("removal")
-    private static final int internalThreadCount = AccessController.doPrivileged(
-        new GetIntegerAction("sun.nio.ch.internalThreadPoolSize", 1));
+    private static final int internalThreadCount =
+        Integer.valueOf(System.getProperty("sun.nio.ch.internalThreadPoolSize", "1"));
```

Comment:
```suggestion
        Integer.getInteger("sun.nio.ch.internalThreadPoolSize", 1);
```

Reply:
Yes, the it should use Integer.getInteger here, same thing in Invoker.java.